examples/example.py

```diff
@@ -3,33 +3,33 @@
 
 import time
 
+from os import sys, path
+from smtpapi import SMTPAPIHeader
+
 if __name__ == '__main__' and __package__ is None:
-    from os import sys, path
     sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
```

Comment:
Was this a change for PEP8?  I think we may need to make sure the sys.path.append comes before importing smtpapi.

Reply:
@misterdorm 
Not completely related to PEP8. Just redefined import order.
But now it showing conflict. I will fix the issue asap.

Reply:
Fixed.
Replace file with Master Branch code.